examples/keymatch_policy.yaml

```diff
@@ -0,0 +1,8 @@
+p:
+  - [alice, /alice_data/*, GET]
+  - [alice, /alice_data/resource1, POST]
+
+  - [bob, /alice_data/resource2, GET]
+  - [bob, /bob_data/*, POST]
+
+  - [cathy, /cathy_data, (GET)|(POST)]
```

Comment:
Please add more tests to verify domain feature is working. You can refer these [test](https://github.com/casbin-rs/sqlx-adapter/tree/master/examples)